schemas/schemaProvider.go

```diff
@@ -10,6 +10,7 @@ func getSchemaMap() map[string]string {
 		"filter":      "configuration/filterImport.json",
 		"design":      "configuration/designImport.json",
 		"publish":     "configuration/publishCatalogItem.json",
+		"helmRepo":    "configuration/helmRepoConnection.json",
```

Comment:
@Yashsharma1911, why is there no adjoining UISchema?

Reply:
For this helmConnection schema there was not much use of it, but as we are maintaining files separately and to maintain consistency I added the ui file